src/Components/ListOfArticles.jsx

```diff
@@ -0,0 +1,23 @@
+import ArticleCard from "./ArticleCard";
+import HomeHeader from "./HomeHeader";
+
+export default function ListOfArticles({ articles }) {
```

Comment:
Can you please use an `isLoading` state to conditionally render something to make sure the user knows the articles are loading?